### PR TITLE
Toc bullet color

### PR DESCRIPTION
**Pull request by AbhijitNage123**
-Added bullet point color option in TOC.

### CODE REVIEW

**Comment by RajkiranBagal** [comment]
@AbhijitNage123 Resolve conflicts

File: classes/class-uagb-block-helper.php [hunk 1]
```diff
@@ -4630,6 +4636,9 @@ public static function get_table_of_contents_css( $attr, $id ) { 			// @codingSt
 				" .uagb-toc__list-wrap > ul.uagb-toc__list > li:first-child" => array(
 				    "padding-top" => 0
 				),
+				" .uagb-toc__list-wrap > ul.uagb-toc__list > li" => array(
+				    "color" => $attr["bulletColor"]
+				),
```
Comment (RajkiranBagal) [inline classes/class-uagb-block-helper.php:4641]: Why have you written this CSS 3 times?

File: src/blocks/table-of-contents/styling.js [hunk 2]
```diff
@@ -218,6 +225,9 @@ function styling( props ) {
 		" .uagb-toc__list-wrap > ul.uagb-toc__list > li:first-child" : {
 		    "padding-top": generateCSSUnit( contentPaddingMobile, contentPaddingTypeMobile )
 		},
+		" .uagb-toc__list-wrap > ul.uagb-toc__list > li" : {
+			"color" : bulletColor
```
Comment (RajkiranBagal) [inline src/blocks/table-of-contents/styling.js:229]: Why have you written this CSS 3 times?

**Comment by vrundakansara** [comment]
1. Capitalize these words - https://share.getcloudapp.com/RBudgDWx
2. Color is not applied to inner bullets - https://share.getcloudapp.com/jkuK9ke8

Fix these issues @AbhijitNage123

**Pull request merged**